packages/lib/src/vis/shared/SelectionTool.tsx

```diff
@@ -2,10 +2,11 @@ import { useKeyboardEvent, useToggle } from '@react-hookz/web';
 import { useThree } from '@react-three/fiber';
 import type { ReactElement } from 'react';
 import { useCallback, useState } from 'react';
+import { useRafState } from 'react-use';
```

Comment:
Weren't you intending to use `react-hookz` version here ?

Reply:
Ouch, well spotted!!